packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/components/container/container.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { SxaComponent } from '../sxa.component';
+
+@Component({
+  selector: 'app-container',
+  templateUrl: './container.component.html',
+})
+export class ContainerComponent extends SxaComponent implements OnInit {
+  placeholderName: string;
+  wrapped: boolean;
+
+  override ngOnInit() {
+    super.ngOnInit();
+
+    this.placeholderName = `container-${this.rendering.params?.DynamicPlaceholderId}`;
+    this.wrapped = this.rendering.params?.Styles?.split(' ').includes('container');
+  }
+
+  get backgroundStyle() {
+    const backgroundImage = this.rendering.params?.BackgroundImage;
+    const mediaUrlPattern = new RegExp(/mediaurl=\"([^"]*)\"/, 'i');
+    if (!backgroundImage || !backgroundImage.match(mediaUrlPattern)) {
+      return {};
+    }
+    const mediaUrl = backgroundImage.match(mediaUrlPattern)[1];
+    return {
+      backgroundImage: `url('${mediaUrl}')`,
```

Comment:
I see that we are missing "prefix" here, like we have in "nextjs-sxa"
Did you check it? See [here](https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Container.tsx#L28)

Reply:
I did, but couldn't figure out why / what this was accomplishing. In all the scenarios I tested, the `BackgroundImage` value looks like this:
```
"BackgroundImage": "<image mediaid=\"d74ba78e-0be0-4414-ac61-7df39d5a1935\" mediaurl=\"/-/media/Feature/JSS-Experience-Accelerator/Basic-Site/my-awesome-image.png\" />"
```
The nextjs implementation constructs a URL using the `mediaid`, but the `mediaurl` gives us a working URL already so I am simply using it. Maybe @matkovskyi has some insight?